CHANGELOG.md

```diff
@@ -1,3 +1,132 @@
+### citus v9.5.11 (April 20, 2023) ###
+
+* Fixes a bug that could break `DROP SCHEMA/EXTENSON` commands when there is a
+  columnar table (#5458)
```

Comment:
I think it's not possible to backport something related to columnar to a version < 10.0 because it's the version that introduces columnar.

As we can see in https://github.com/citusdata/citus/pull/5458, it's only backported to 10.0, 10.1 & 10.2.